agnostic-svelte/src/lib/components/Input/Input.svelte

```diff
@@ -370,18 +370,16 @@ borders that visually conflict. */
   };
   
   $: addonContainerClasses = () => "input-addon-container";
-
 </script>
 <div class="w-100">
   <label class={labelClasses} for={id}>{label}</label>
   {#if type == "textarea"}
     <textarea
       id={id}
-      bind:value
       class={inputClasses}
       on:blur
       on:change
-      on:input
+      bind:value
```

Comment:
Great feedback @longrunningprocess !

[147-two-way-binding-spike d4db9b3f] Use bind:value shorthand for textarea since not dynamically using type there